frameworks/person-escort-record/questions/concealed-items.yml

```diff
@@ -12,6 +12,10 @@ options:
         -
           type: required
           message: Enter details of weapons the person has created or used
+    flags:
+      -
+        label: Created/used weapons
```

Comment:
`Created or used`? GOV.UK styleguide warns against [using slashes in place of or](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#or).